dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -2048,6 +2048,39 @@ def test_null_replication_metrics_calculation(self):
         np.testing.assert_array_almost_equal([17 / 8, 48 / 8], column["class_mean"][0])
         np.testing.assert_array_almost_equal([12 / 2, 6 / 2], column["class_mean"][1])
 
+        # Test with all null in a column
+        data_3 = pd.DataFrame([[9999999, 9], [9999999, 9]])
```

Comment:
in the future, might be worthwhile to create a dataset with more edge cases 
`data_3 = pd.DataFrame([[9999999, 9999999, 123], [9999999, 123, 123]])`